src/Rvig.HaalCentraalApi.Personen/Services/GbaPersonenApiService.cs

```diff
@@ -194,9 +271,23 @@ await LogProtocolleringInDb(afnemerCode, result.Select(x => x.pl_id).ToList(),
 
             var bsns = personenPlIds.Select(p => p.persoon.Burgerservicenummer).Where(bsn => !bsn.IsNullOrEmpty()).ToList();
 
-            IEnumerable<GbaPersoon> persoonGezagsrelaties = personenPlIds.FirstOrDefault().persoon is GbaGezagPersoonBeperkt
+            IEnumerable<Persoon> persoonGezagsrelaties = personenPlIds.FirstOrDefault().persoon is GbaGezagPersoonBeperkt
```

Comment:
De code voor het bepalen van gezag voor een beperkt persoon is praktisch een duplicaat van de code voor een persoon. Extracten naar een method om duplicatie weg te werken?

Reply:
Apart dat Sonar dit niet heeft gedetecteerd